docker/test/integration/minifi/processors/RelativeImporterProcessor.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from ..core.Processor import Processor
```

Comment:
Ah what a handy feature including relative imports. Can this also be done in MiNiFi Python binding processors or mainly NiFi Native Python processors?

Reply:
The feature was focused on the NiFi python processor support, but this implementation should work with the MiNiFi style python processors as well. Feel free to try it out and report back any problems :)